app/javascript/dashboard/components/widgets/conversation/ReplyEmailHead.vue

```diff
@@ -67,15 +67,17 @@ export default {
   data() {
```

Comment:
Can probably remove the props object, since they're not being used in the component